openshift/catalog-sync/test/suite_test.go

```diff
@@ -0,0 +1,52 @@
+package test
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github/operator-framework-operator-controller/openshift/catalog-sync/pkg/check"
+	"github/operator-framework-operator-controller/openshift/catalog-sync/pkg/extract"
+)
+
+func TestSuite(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Catalog Validation Suite")
+}
+
+// images contains the list of images to be validated.
+var images = []string{
+	"registry.redhat.io/redhat/community-operator-index:v4.18",
+	"registry.redhat.io/redhat/redhat-marketplace-index:v4.18",
+	"registry.redhat.io/redhat/certified-operator-index:v4.18",
+	"registry.redhat.io/redhat/redhat-operator-index:v4.18",
+}
+
+var _ = Describe("Catalog Image Validation Suite", func() {
+	for _, url := range images {
+		name := getImageNameFrom(url)
```

Comment:
Is there some reason we need to do the name remapping? It could be useful if the entire ref shows up in the test name.